libcontainer/specconv/spec_linux.go

```diff
@@ -190,6 +190,11 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 	exists := false
 	for _, m := range spec.Mounts {
+		if m.Type == "bind" || m.Type == "rbind" {
+			if len(m.Options) == 0 {
+				return nil, fmt.Errorf("Should add options when mount type is (r)bind.")
```

Comment:
/s/"Should add options when mount type is (r)bind."/"Must include options when mount type is (r)bind for: %v", m/

Reply:
should probably have a test case for this

Reply:
and it's not an error .. should be a warning

Reply:
the runtime spec points to the linux mount manual for details. 

A read of the mount man shows that we should probably be using the mount options of the source if not specified. Thus I would not expect an error to the thrown, that's probably a bug in our implementation.

Reply:
We have always defered a lot of things to the underlying syscall.  We cannot replicate every error from ever syscall so the strategy has been, call mount with no options and it will fail anyways.

Reply:
that makes sense.. so if the syscall is "improved" then we'll just naturally pick that up

Reply:
yes, or else it would be a never ending battle trying to catch up or stay in sync with the underlying system

Reply:
> /s/"Should add options when mount type is (r)bind."/"Warning no options specified for(r)bind mount: %v", m/

Changed.

> should probably have a test case for this

Yes, I have add it.